test/amoc_api_scenario_handler_SUITE.erl

```diff
@@ -59,17 +57,6 @@ end_per_testcase(
 end_per_testcase(_, _Config) ->
     destroy_env().
 
-get_scenario_status_returns_200_when_scenario_exists(_Config) ->
```

Comment:
Why this test case was removed?

Reply:
it's a very old one. `given_test_status_mocked(true)` - this is completely invalid, the mocked function should never return `true` value (even before the migration), `loaded` is used instead.